### ISSUE TITLE
Can you please make it work with null-safety

### ISSUE THREAD

**Issue by uc-dve** [vanelizarov/flutter_macos_webview#7]
Can you please make it work with null-safety

**Comment by Jesperpaulsen** [comment]
I created a PR for null safety now. Until it's merged you can use my [fork](https://github.com/Jesperpaulsen/flutter_macos_webview.git), by replacing the package in pubspec.yaml:

```
flutter_macos_webview:
    git:
      url: 'https://github.com/Jesperpaulsen/flutter_macos_webview.git'
```

**Comment by MrHazee** [comment]
Nice @Jesperpaulsen ! I get 

> "Error: MissingPluginException(No implementation found for method open on channel com.vanelizarov.flutter_macos_webview/method)" 

when I try to open a wensite. Any idea?
Thanks

**Comment by Jesperpaulsen** [comment]
That's strange @MrHazee. What Flutter version are you using? I'm a bit busy atm, but will try to look into this in the coming days.

**Comment by weifengsonos** [comment]
Hi @Jesperpaulsen,

I am using Flutter SDK "2.2.3-stable" and seeing the error below when clicking one of the buttons ("Open as modal" or "Open as sheet") in the demo app.  Have you seen this in your null-safety plugin?

flutter: Error: 2, WebResourceErrorType.webContentProcessTerminated, WKErrorDomain, Error Domain=WKErrorDomain Code=2 "(null)"

**Comment by vanelizarov** [comment]
Published 0.0.5 with null-safety support
Thanks @Jesperpaulsen!
